SECURITY.md

```diff
@@ -0,0 +1,51 @@
+# Security Policy
+
+## Overview
+
+We value the contributions of the community and welcome any input on potential security issues.
+To ensure the safety of our users, we encourage following responsible disclosure practices described in this document.
+
+## Supported Versions
+
+Only the latest release version is supported.
+We use SemVer and encourage to pin only to the major version of our software and update to the latest minor and patch versions regularly.
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability in our repository, we encourage you to report it to us as quickly as possible.
+Please do not publicly disclose the details of the vulnerability until we have had a chance to address it.
+
+### How to Report
+
+1. **Email**:
+   Send an email with vulnerability description to <opensource@reef.pl> .
+
+   Please use the following template:
+
+   > ## Description of the vulnerability
+   >
+   > ## Steps to reproduce
+   >
+   > ## Potential impact
+   >
+   > ## Any potential fixes or mitigations
+   >
+   > ## How would you like to be attributed in the public changelog
+   >
+   > e.g., name, email, or GitHub handle
+
+2. **Response Time**:
+   We will acknowledge your report within 30 days and provide a timeline for fixing the vulnerability.
```

Comment:
We can do much better than that. I can't imagine a situation where we'd need more than a week, 10 days tops